lib/src/kvs/ds.rs

```diff
@@ -37,6 +37,12 @@ pub struct Datastore {
 	notification_channel: Option<(Sender<Notification>, Receiver<Notification>)>,
 }
 
+impl fmt::Debug for Datastore {
+	fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+		f.write_str("Datastore { .. }")
```

Comment:
This could be replaced with `f.debug_struct("Datastore").finish_non_exhaustive()` for consistency.